library/ssl_tls.c

```diff
@@ -6549,7 +6546,8 @@ static int ssl_session_load( mbedtls_ssl_session *session,
         if( 8 > (size_t)( end - p ) )
             return( MBEDTLS_ERR_SSL_BAD_INPUT_DATA );
 
-        start = ( (uint64_t) p[0] << 56 ) |
+        session->start =
```

Comment:
The `(time_t)` conversion went missing here - can you reintroduce it?

Reply:
Okay